app/api.ts

```diff
@@ -1,4 +1,5 @@
-import { JoinFormInput, JoinFormResponse, NNAssignFormInput, NNAssignFormResponse, QueryFormInput, QueryFormResponse } from "@/app/io";
+"use client"
```

Comment:
Is this a typo?

Reply:
Nope.

Reply:
What does it do?

Reply:
forces the API calls to happen on the client. I had some trouble where it was behaving as if it was happening serverside, and that made it impossible to test (I spent literally all of saturday trying to make it work and got nowhere) so I gave up and went to this instead. Currently, the expected behavior is that the client is calling meshdb anyway.